examples/xx-delete-editor-layout-field-set.js

```diff
@@ -0,0 +1,7 @@
+// TODO: rename file before merging to master
+
+module.exports = function (migration) {
+  const Page = migration.editContentType('page');
```

Comment:
Nit: I think in other examples we're using lower-cased identifiers to indicate that they are instances rather than classes.